scripts/irods/convert_configuration_to_json.py

```diff
@@ -61,8 +61,11 @@ def get_env_file_path(env_file):
         fullpath = '/var/lib/irods/.irods/' + env_file
         return fullpath
 
+# TODO Review this
```

Comment:
Investigate

Reply:
@trel Can this entire file be removed? I don't see any reference to it in the repo. It also appears to reference names like `irodsHost` which feels pre-4.2.

Reply:
I think I found it! https://github.com/irods/irods/commit/a953c027ec93b24ee3ecfe409d4da244e278c1e9

Looks like we can remove this.

Any objections @trel?